README.md

```diff
@@ -145,11 +145,3 @@ See https://nx.dev/features/manage-releases for how to use `nx release` to:
 - Update version number across all projects;
 - Auto-update the CHANGELOG;
 - Release the new version of all projects.
-
-## ToDo: Set up CI!
```

Comment:
FYI you could link to the `DEV_TO_DO.md` here if you want it to have some visibility.